src/_nested-list.scss

```diff
@@ -32,6 +32,8 @@ $button: #{$fd-namespace}-button;
     }
   }
 
+  $border-radius: 0.25rem;
```

Comment:
That's not the way to declare a variable, please check other components how it's done

Reply:
using `$fd-nested-list-popover-border-radius` now.